sycl/source/detail/windows_pi.cpp

```diff
@@ -0,0 +1,12 @@
+#include <windows.h>
+#include <winreg.h>
+#include <string>
+
+void *loadOsLibrary(const std::string &PluginPath) {
+  // TODO: Check if the option RTLD_NOW is correct.
```

Comment:
Copy-and-paste comment.

Wondering if the path separators need to be changed to be Windows compatible (should the `PluginPath` contain Unix forward slashes)?

Reply:
Currently, we do not have this problem , since we only use the name of the file in PluginPath, and the file is looked for in paths listed in LD_LIBRARY_PATH(Unix) or LIB(Windows) EV. 
After we extend this logic, maybe we will need to do that.

Reply:
Deleted the comment.